src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,25 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
+
+public class Deadline extends Task {
+    public String dl;
```

Comment:
The naming of this field is quite unclear. Perhaps, you can try naming it with more descriptive words instead of just letters.